Bottom: 8a37bb5b752570cd64b1bdb2562cf2182fb6c357
Top:    f25c5c2d2c37cf8acaf74616a979821c0b59707f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 22:28:55 +0000

Refresh of more-changes

---

diff --git a/src/polygon1.c b/src/polygon1.c
index ef11dd8..a17cff8 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3407,8 +3407,7 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
       /* intersect needs to make a list of the contours in a and b which are intersected */
       M_POLYAREA_intersect (&e, a, b, TRUE, &the_list);
 
-      /* XXX - Need to loop the intersection routines until the geometry stabalises...
-       *       Adding an extra round of intersection fixes at least one case here
+      /* XXX - Need to loop the intersection routines until the geometry stabalises???
        */
 //      M_POLYAREA_intersect (&e, a, b, TRUE, &the_list);
